.evergreen/config.yml

```diff
@@ -511,12 +511,19 @@ functions:
           export CSFLE_SHARED_LIB_PATH=$(.evergreen/find-crypt_shared.sh "$PROJECT_DIRECTORY/crypt_shared/lib/")
           export DISABLE_CRYPT_SHARED=${DISABLE_CRYPT_SHARED}
           export TLS_FEATURE=${TLS_FEATURE}
+          export PYTHON=${PYTHON}
+          ${PYTHON} -m pip install boto3
+
           # Exported without xtrace to avoid leaking credentials
           set +o xtrace
-          export KMS_PROVIDERS=$(cat << "EOF"
-          ${kms_providers}
-          EOF
-          )
+          export AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
```

Comment:
These need to be individual rather than a single json blob because the aws temporary credentials script needs the aws parts of this.  I also realized it's better to do it this way so that individual credentials can be easily rotated.